src/vs/base/browser/markdownRenderer.ts

```diff
@@ -211,9 +211,12 @@ export function renderMarkdown(markdown: IMarkdownString, options: MarkdownRende
 	);
 
 	function filter(token: { tag: string, attrs: { readonly [key: string]: string } }): boolean {
-		if (token.tag === 'span' && markdown.isTrusted) {
-			if (token.attrs['style'] && Object.keys(token.attrs).length === 1) {
+		if (token.tag === 'span' && markdown.isTrusted && (Object.keys(token.attrs).length === 1)) {
+			if (token.attrs['style']) {
 				return !!token.attrs['style'].match(/^(color\:#[0-9a-fA-F]+;)?(background-color\:#[0-9a-fA-F]+;)?$/);
```

Comment:
This regex is too restrictive to allow anything other than `#FF00FF` style colors, would be nice to support `rgb()` etc